crates/cxx-qt/include/threading.h

```diff
@@ -32,11 +31,20 @@ class CxxQtThreading : public CxxQtLocking
 
   CxxQtThread<T> qtThread() const
   {
-    return CxxQtThread<T>(m_cxxQtThreadObj, m_rustObjMutex);
+    // Note that this assumes that T inherits from CxxQtLocking, which we assert
+    // during parsing.
```

Comment:
do we add a `Q_ASSERT` to confirm this?

Reply:
Ah, Locking is gone, so that comment needs to go as well.